src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/Coverage.java

```diff
@@ -29,7 +30,7 @@
  *     <li><b><a href="https://www.broadinstitute.org/gatk/guide/tooldocs/org_broadinstitute_gatk_tools_walkers_annotator_DepthPerSampleHC.php">DepthPerSampleHC</a></b> calculates depth of coverage after filtering by HaplotypeCaller.</li>
  * </ul>
  */
-public final class Coverage extends InfoFieldAnnotation {
+public final class Coverage extends InfoFieldAnnotation implements StandardAnnotationKind {
```

Comment:
Are you marking the same annotations as `Standard` that were marked as such in GATK3?